src/components/metadata/metadata.test.js

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
 import MetaData from './index';
 import { getClickedNodeMetaData } from '../../selectors/metadata';
-import { setup, mockState } from '../../utils/state.mock';
-import { addEdgeLinks } from '../../utils/graph/graph';
+import { toggleNodeClicked } from '../../actions/nodes';
+import { setup, prepareState } from '../../utils/state.mock';
+import animals from '../../utils/data/animals.mock.json';
 
 const salmonTaskNodeId = '443cf06a';
 const catDatasetNodeId = '9d989e8d';
 const rabbitParamsNodeId = 'c38d4c6a';
 
 describe('MetaData', () => {
-  // Add edge links, can be removed when new graph is default
-  addEdgeLinks(mockState.animals.graph.nodes, mockState.animals.graph.edges);
```

Comment:
I think the new graph is currently loaded by default?

In case for some reason it's not loaded in the tests, instead of adding this pending removal for the oldgraph flag, this should ideally not needed by specifying `{flag: {oldgraph: false}}` within the `prepareState` method on line 35 here?

Reply:
@bru5 has done this bit of changes and prepareState. I am not sure of the above -- can we get on a call to discuss?

Reply:
@studioswong That used to be the case but as the line in question is being removed here I don't think it's an issue?